packages/svelte2tsx/src/svelte2tsx.ts

```diff
@@ -789,10 +786,17 @@ function processInstanceScriptContent(str: MagicString, script: Node): InstanceS
     pendingStoreResolutions.map(resolveStore);
 
     // declare implicit reactive variables we found in the script
-    for (const [name, {pos, expr}] of implicitTopLevelNames.entries()) {
+    for (const [name, pos] of implicitTopLevelNames.entries()) {
         if (!rootScope.declared.has(name)) {
-            const exprStr = str.slice(astOffset + expr.getStart(), astOffset + expr.getEnd());
-            str.prependRight(pos + astOffset, `;let ${name} = ${exprStr}; `);
+            // remove '$:' label
+            str.remove(pos + astOffset, pos + astOffset+2);
+            const followingChar = str.slice(pos + astOffset+2, pos + astOffset+3);
+            // if there exists whitespace after '$:', we do not insert more space.
```

Comment:
Not sure if we need this. Is there any advantage over just doing it either way?

Reply:
we dont care about the format of the generated code？

Reply:
We don't need to, we can do all the bad formatting we want :-P